clappr/clappr-ios
# Remove container layer

## 🏁 Goal
Since the Container is a wrapper around the playback and Every playback is associated with a container,  the goal is to join the ContainerLayer and the PlaybackLayer of the LayerComposer into only one layer.

## ✅ How to test

1. Run the example
1. Enable view hierarchy debugger
1. You should see the following layers from the LayerCompose in the layer hierarchy. 
 - backgroundLayer
 - playbackLayer
 - mediaControlLayer
 - coreLayer
 - overlayLayer
  
## 🖼 Screenshots
![image](https://user-images.githubusercontent.com/15603892/100451988-0e1e6500-3097-11eb-9a54-cb244b5447c9.png)
